src/modules/charge-versions-upload/lib/validator/index.js

```diff
@@ -114,12 +116,32 @@ const validateGroups = async (rows, headings, jobName) => {
   const getStartDate = row => row[headings.indexOf('charge_information_start_date')]
 
   logger.info(`${jobName}: sorting rows`)
-  const sortedRows = sortBy(rows.map((row, index) => {
+
+  const mapped = rows.map((row, index) => {
     const licenceNumber = row[headings.indexOf('licence_number')]
     const chargeReferenceDetailsChargeElementGroup = row[headings.indexOf('charge_reference_details_charge_element_group')]
     const rowNumber = index + ROW_OFFSET
+
     return { licenceNumber, chargeReferenceDetailsChargeElementGroup, rowNumber, row }
-  }), ['licenceNumber', 'chargeReferenceDetailsChargeElementGroup'])
+  })
+
+  // Sorts the rows based on the licenceNumber and the chargeReferenceDetailsChargeElementGroup
+  const sortedRows = mapped.sort((a, b) => {
+    if (a.licenceNumber < b.licenceNumber) {
+      return -1
+    } else if (a.licenceNumber > b.licenceNumber) {
+      return 1
+    } else {
+      if (a.chargeReferenceDetailsChargeElementGroup < b.chargeReferenceDetailsChargeElementGroup) {
+        return -1
+      } else if (a.chargeReferenceDetailsChargeElementGroup > b.chargeReferenceDetailsChargeElementGroup) {
+        return 1
+      } else {
+        return 0
+      }
+    }
+  })
```

Comment:
In `src/modules/charge-versions-upload/lib/mapper/index.js` you opted to do the same thing by creating a new `sortBy()` method and using ternary `if` statements to reduce the nesting.

Of the 2 approaches I think it was clearer. So I would suggest seeing if there is a way to refactor this change to use the same approach. What do you think? 🤷